styles/components/badges/styles.less

```diff
@@ -0,0 +1,450 @@
+& when (@badge-enabled) {
+  /* Badge */
+
+  .badge {
+    align-items: center;
+    display: inline-flex;
+    justify-content: center;
+    .element-shape-style(badge);
+    .element-spacing(badge, null, null);
+    .element-text-size(badge, null);
+    .element-text-style(badge);
+    .property-variant(badge, border-radius, null, null);
+
+    /* Layout Variants */
+
+    &.badge-narrow {
+      .element-spacing(badge, narrow, null);
+    }
+
+    &.badge-wide {
+      .element-spacing(badge, wide, null);
+    }
+
+    & > * {
+      display: inline-flex !important;
```

Comment:
I guess I'm just not seeing why children need the `inline-flex` property. Can you expand on that?